openfisca_core/formulas.py

```diff
@@ -925,7 +939,7 @@ def new_filled_column(__doc__ = None, __module__ = None,
     return column
 
 
-def set_input_dispatch_by_period(formula, period, array):
+def set_input_dispatch_by_period(formula, period, array, behavior=None):
```

Comment:
That's probably because all the `set_input***` must have the same signature, and the extra argument is only needed by `set_input_divide_by_period `.